### ISSUE TITLE
Centos8 error: "symbol lookup error: undefined symbol: argon2_ctx"

### ISSUE THREAD

**Issue by orbitlens** [ranisalt/node-argon2#405]
I'm having problems running this package on Centos8 server. 
Error details: 
```
/srv/api/src/index.js: symbol lookup error: /srv/stellar-trace-api/node_modules/.pnpm/argon2@0.40.1/node_modules/argon2/build/Release/argon2.node: undefined symbol: argon2_ctx
```

### Environment
argon2 version: `0.40.1`
OS: `Centos8`
Node: `v20.2.0`
Package manager: `pnpm`

I tested the code locally and it seems to be working without any issues on different operation systems. Tried to force rebuild the package with the command `pnpm rebuild argon2` but it doesn't seem to actually run the sources compilation process and uses the pre-built binary instead.

With `0.31.2` everything works without any problems on the same environment.

**Comment by ranisalt** [comment]
Can you share a reproduction? Ideally within a Docker container, you can use the `centos:centos8` image as a base.

With that said, considering CentOS 8 reached end-of-life [over 2 years ago](https://www.centos.org/centos-linux-eol/) it's a bit hard to provide support - you really shouldn't be using it (dismiss the info if you're on CentOS Stream 8 instead)

You can force building from source on NPM setting the environment variable `npm_config_build_from_source=true`, try that with pnpm

**Comment by ranisalt** [comment]
#415 should fix this issue, can you try with v0.41.0 or newer?
